quickjs.c

```diff
@@ -20810,8 +20810,8 @@ static int __exception js_parse_property_name(JSParseState *s,
             if (next_token(s))
                 goto fail1;
             if (s->token.val == ':' || s->token.val == ',' ||
-                s->token.val == '}' || s->token.val == '(') {
-                is_non_reserved_ident = TRUE;
+                s->token.val == '}' || s->token.val == '(' ||
+                s->token.val == '=' ) {is_non_reserved_ident = TRUE;
```

Comment:
Missing line break.
Suggested style:
```
                s->token.val == '}' || s->token.val == '(' ||
                s->token.val == '=' ) {
                    is_non_reserved_ident = TRUE;
```

Reply:
Already solved, it was copied as it was over from the source material but later fixed.